main.py

```diff
@@ -95,6 +96,28 @@ async def get_experiment_info(file: str) -> Any:
     return remote.examine(file)
 
 
+previous_queue = {}
+current_queue = {}
+@app.get("/experiment/queue/", response_model=Dict[int, Dict[str, Any]])
+async def get_experiment_queue() -> Dict[int, Dict[str, Any]]:
+    """Gets the list of queued experiment and returns it.
+
+    Returns:
+        A dictionary of queued experiments with rid as their keys.
+        The value of each corresponding rid is a dictionary with several information:
+          "priority", "status", "flush", "pipeline", "expid", and etc.
```

Comment:
It is a minor suggestion, but please remove "and".
The "etc" includes the meaning of "and".

Reply:
There is the `timed` (or due date) key, right?

I think we need the information, so it looks better to include this key in the above docstring!
And it is okay to remove "fluse" as we probably won't use it.